tests/code_size/hello_webgl_wasm.json

```diff
@@ -1,10 +1,10 @@
 {
   "a.html": 563,
   "a.html.gz": 377,
-  "a.js": 4549,
-  "a.js.gz": 2240,
-  "a.wasm": 10917,
-  "a.wasm.gz": 6935,
-  "total": 16029,
-  "total_gz": 9552
+  "a.js": 4651,
```

Comment:
what causes the increase here?

Reply:
I think this is the JS side code in `dynCall` and `makeDynCaller`.   The wasm gets smaller in proportion to the number of signatures and the JS grows by a fixed amount I think.

In summary:
1. Small JS size increase.
2. Only paid by programs that use dynCalls.
3. Should be removable once we can switch 100% to table calls.

Reply:
-1 for this change: the old code is both smaller and more performance, as it avoids dynamic signature binding. I think we should switch to this only after we have step 3. from above in-place in this same PR. That way we will not have to accept a regression on a promise of things improving back to old form later on.

Reply:
OK that makes sense.  I'll keep going with this until its all done rather then try to land this incremental change.